lib/nightmare.js

```diff
@@ -67,6 +71,14 @@ function Nightmare(options) {
     stdio: [null, null, null, 'ipc']
   });
 
+  this.proc.stdout.on('data', (data) => {
```

Comment:
It’s unlikely to be too big a deal, but as a general practice, it’d be a good idea to pipe through `split` or `split2` first—there’s no guarantee otherwise that stuff will come out of the `stdout/stderr` streams as whole lines (could be more or less).